lib/ios/AirMaps/AIRMapManager.m

```diff
@@ -55,18 +57,26 @@ - (UIView *)view
     
     // MKMapView doesn't report tap events, so we attach gesture recognizers to it
     UITapGestureRecognizer *tap = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(handleMapTap:)];
+    UITapGestureRecognizer *doubleTap = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(handleMapDoubleTap:)];
+    [doubleTap setNumberOfTapsRequired:2];
+    [tap requireGestureRecognizerToFail:doubleTap];
+    
     UILongPressGestureRecognizer *longPress = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(handleMapLongPress:)];
     UIPanGestureRecognizer *drag = [[UIPanGestureRecognizer alloc] initWithTarget:self action:@selector(handleMapDrag:)];
     [drag setMinimumNumberOfTouches:1];
     [drag setMaximumNumberOfTouches:1];
     // setting this to NO allows the parent MapView to continue receiving marker selection events
     tap.cancelsTouchesInView = NO;
+    doubleTap.cancelsTouchesInView = NO;
     longPress.cancelsTouchesInView = NO;
-
+    
+    doubleTap.delegate = self;
+    
     // disable drag by default
     drag.enabled = NO;
-
+    
     [map addGestureRecognizer:tap];
+    [map addGestureRecognizer:doubleTap];
```

Comment:
I wanted to shuffle around the above code - I think it would make more sense to initialise and configure them type-wise rather than operation-wise, but I didn't want to make the review harder than it has to be.